test/interception.spec.js

```diff
@@ -634,8 +643,6 @@ module.exports.describe = function({testRunner, expect, defaultBrowserOptions, p
   describe('ignoreHTTPSErrors', function() {
     it('should work with request interception', async({newPage, httpsServer}) => {
       const page = await newPage({ ignoreHTTPSErrors: true, interceptNetwork: true });
-
-      await page.setRequestInterception(true);
```

Comment:
Why this change?

Reply:
This is redundant. `interceptNetwork: true` above has the same effect.